src/components/Map/Map.js

```diff
@@ -24,7 +24,9 @@ const Wrapper = styled.main`
   }
 
   @media (max-width: 960px) {
-    width: 90vw;
+    width: 100vw;
+    margin: 0;
+    z-index: -1;
```

Comment:
This `z-index: -1` is hidden the map below the container flex div (and all other elements, it is the reason because @Israel-Laguan and I can't use the map in mobile)

Look how in the image a element is over the map
![imagen](https://user-images.githubusercontent.com/66505715/140073992-dca97de6-0adc-4fc7-97a0-a322e214c3e9.png)

Reply:
I have tried and this work: Instead of use `z-index: -1` to the map (delete this property, because all other items will be over the map and then we can't click or use the map), please use `z-index: 1` on the div that you want to be up.

Look that in the image, I only have added the z-index property to this container (the header), then, the header is over the map and I can still use the map
![imagen](https://user-images.githubusercontent.com/66505715/140074350-2ea6543e-4653-40b0-b195-d8d219be4422.png)